crates/oxc_linter/src/snapshots/iframe_missing_sandbox.snap

```diff
@@ -0,0 +1,97 @@
+---
+source: crates/oxc_linter/src/tester.rs
+---
+  ⚠ eslint-plugin-react(iframe-missing-sandbox): An iframe element is missing a sandbox attribute
+   ╭─[iframe_missing_sandbox.tsx:1:2]
+ 1 │ <iframe></iframe>;
+   ·  ──────
+   ╰────
+  help: Add a `sandbox` attribute to the `iframe` element.
```

Comment:
Could we provide an auto-fixable suggestion? If so, can you either 1) implement it, or 2) add a `pending` fix kind?